index.html

```diff
@@ -309,6 +309,34 @@ <h3>
   </p>
 </section>
 
+<section> <h2>Pressure Factors</h2>
+  <p>
+    <dfn>Pressure factors</dfn> represents factors affecting system performance and current [=pressure state=].
+    In case the [=pressure state=] is nominal, the {{ComputePressureRecord.factors}} will always an [=list/empty=]
+    sequence.
+  </p>
+  <aside class="note">
+    The [=user agent=] might not be able to expose the factors for all systems or for all [=pressure states=].
+  </aside>
+  <pre class="idl">
+    enum ComputePressureFactor { "thermal", "power-supply" };
+  </pre>
+  <p>
+    The <dfn>ComputePressureFactor</dfn> enum represents the [=pressure factors=]:
+  </p>
+  <p>
+    <ul>
+      <li>
+        <b>Thermal</b>: The system is under elevated thermal conditions resulting in the system being
```

Comment:
I don't like the formulation, but I don't have better :)
The "resulting  in" disturbs me.